README.md

```diff
@@ -74,8 +74,9 @@ https://cn.dapr.io/meeting/
 
 Every week, we meet to plan and sync on the progress for the milestone's release. This meeting includes maintainers and active contributors and usually lasts for about 30-60 minutes. If you would like to **actively contribute** towards a milestone, you are welcome to join that meeting too.
 
-**Zoom Call:** https://zoom.us/j/92337656846<br>
-**Passcode:** 815623<br>
+**Zoom Call:** https://zoom.us/j/91940016938<br>
+**Meeting ID:** 919 4001 6938<br>
+**Passcode:** 152697<br>
```

Comment:
I am keeping the passcode but if we have zoom-bombing incidents again, we will need to generate a new code and keep it hidden and only given on-demand, like before.